projects/igniteui-angular/src/lib/input-group/input-group.component.spec.ts

```diff
@@ -178,26 +178,50 @@ describe('IgxInputGroup', () => {
         expect(inputGroupElement.classList.contains(INPUT_GROUP_COMPACT_DENSITY_CSS_CLASS)).toBeTruthy();
     });
 
-    it('suppress focus on input when clicked', async () => {
-        const fixture = TestBed.createComponent(InputGroupSupressInputFocusComponent);
+    it('should not fire focus and blur on prefix or suffix click', () => {
+        const fixture = TestBed.createComponent(InputGroupComponent);
         fixture.detectChanges();
 
         const inputGroup = fixture.componentInstance.igxInputGroup;
-        UIInteractions.clickElement(inputGroup.element);
-        await wait();
-        fixture.detectChanges();
+        const prefix = fixture.componentInstance.prefix;
+        const input = fixture.componentInstance.igxInput;
+
+        const pointOnPrefix = UIInteractions.getPointFromElement(prefix.nativeElement);
+        const pointerEvent = UIInteractions.createPointerEvent('pointerdown', pointOnPrefix);
+        const preventDefaultSpy = spyOn(pointerEvent, 'preventDefault');
+
+        Object.defineProperty(pointerEvent, 'target', { value: input.nativeElement });
+        inputGroup.onPointerDown(pointerEvent);
```

Comment:
I actually prefer if you're using fixture to go ahead and use `triggerHandler` instead - that also verifies the correct host binding is active. You should be able to pass the event argument with spy function for `preventDefault` just the same.

Reply:
yeah much better in this way - done